metricbeat/module/system/entropy/_meta/data.json

```diff
@@ -0,0 +1,22 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "event": {
+        "dataset": "system.entropy",
+        "duration": 115000,
+        "module": "system"
+    },
+    "metricset": {
+        "name": "entropy"
+    },
+    "service": {
+        "type": "system"
+    },
+    "system": {
+        "entropy": {
+            "entropy": {
```

Comment:
Do we want this double `entropy` level?

Reply:
ups, I missed this when we talked about object vs group, +1 to one level only